newIDE/app/src/UI/VerticalTabButton.js

```diff
@@ -69,7 +69,7 @@ const VerticalTabButton = ({
       {!hideLabel && (
         <>
           <Spacer />
-          {label}
```

Comment:
should we add a tiny comment here for future reference? something like "the span is required to avoid Google Translate crashing the app - link to issue"

Reply:
+1